lib/core/environ.nit

```diff
@@ -46,10 +46,12 @@ redef class String
 	# Search for the program `self` in all directories from `PATH`
 	fun program_is_in_path: Bool
 	do
+		var sep = if is_windows then ";" else ":"
```

Comment:
The path separator seems to me a useful thing to expose as it own function or constant (as it is in most cross-platform languages).